faq/ppv.php

```diff
@@ -20,8 +20,8 @@
 <h2>Contents</h2>
 
 <p>
-<a href="#gen">A) General Procedures</a><br>
-<a href="#rr">B) Requests to Revise and Resubmit</a><br>
+<a href="#general">A) General Procedures</a><br>
+<a href="#revise">B) Requests to Revise and Resubmit</a><br>
```

Comment:
do we have links to these anywhere? Wiki? Would we expect people to have bookmarked these?

Reply:
The `id`s have been added in the wiki documents. @cpeel recommended removing the dummy links that I added to the documents in the code -- which I have done, but just hadn't gotten the squashed commit pushed yet. Just about to do that.

Reply:
Actually, those stayed, because I renamed existing IDs for more clarity. Looks like I took out an extra space in `faq/post_proof.php`, which winds up being the only change that persisted. Some days. <sigh>

`faq/post_proof.php` and `faq/ppv.php` are the old PP and PPV FAQ/Guideline documents, and are way out of date. They've been useful as a means to redirect to wiki content.